### DISCUSSION TITLE
тесты

### DISCUSSION THREAD

**Discussion by Pigzinzspace** [godot-engine-ru/shooter-3d#21] [Ideas]
**содержание документа**  - список тестов которые нужно провести

**Comment by me2beats** [comment]
**тест карты/территории завода**
3D территория, по которой будет бегать игрок — внутренние строения завода, соединённые между собой переходами и подземными ходами.
По геймплею, бегает игрок только по этому пространству, на воздух не выходит
приветствуется территория с примерным расположением комнат, цехов, переходов и тд (всё можно на кубах/примитивах), а не просто стены.

Предпочтителен демо-проект в Годо

**Reply by Pigzinzspace** [reply]
тест думаю лучше делать с полностью  открытими дверями

**Comment by me2beats** [comment]
**Тест демки миссии, спавна, атаки врагов (зомби и тд), ХП, ​движения игрока по карте и сохранения**

очень желательно на демке завода (см выше, пока не сделана).

Надо прицениться, какие варианты и сочетания будут удачными
— конечный или бесконечный спавн
— количество врагов в разных точках
— атака врагов (урон), скорость передвижения
— количество заданий

Это демка, все делаем упрощённо
— оружие у героя может быть пока одно, пистолет (параллелепипед)
— зомби - куб или капсула, с направлением глаз (в какую сторону смотрит)
— атака зомби  может быть пока только ближняя (этого достаточно) , атака наносит дамаг, снимая ХП
— задания могут быть дойти до пункта х
— сохранения достаточно только одного слота и кнопок загрузить, возможно сохранить

**Reply by Pigzinzspace** [reply]
стоит наверно совместить это с тем что уже абдурахман выкладывал
